src/components/core/geoChart/tests/geoChartDataLayers.spec.ts

```diff
@@ -38,6 +39,84 @@ describe("createPushpinDataLayer", () => {
         ]);
     });
 
+    it("should return default border, color and size when Color, Size contains same values", () => {
+        const geoData: IGeoData = {
+            size: {
+                index: 0,
+                name: "size",
+            },
+            color: {
+                index: 1,
+                name: "color",
+            },
+            location: {
+                index: 0,
+                name: "location",
+            },
+        };
+        const excutionResult: Execution.IExecutionResult = {
+            data: [Array(3).fill(1), Array(3).fill(10)],
```

Comment:
the length of these items should be equal, right?
`[Array(3).fill(10), Array(3).fill(10)],`

Reply:
They are two separate arrays, one for Size and one for Color.

Reply:
Yes. the length of these items is 3

Reply:
my bad :)